shared/fs/index.stories.js

```diff
@@ -113,6 +115,61 @@ const load = () => {
     .add('Preview', () => (
       <NormalPreview routePath={I.List([])} routeProps={I.Map({path: '/keybase/private/foo/bar.jpg'})} />
     ))
+    .add('Footer Cards', () => (
+      <Box>
+        <Box style={{height: 8}} />
+        <Download
+          filename="fjweio"
+          completePortion={0.42}
+          progressText="4 s"
+          isDone={false}
+          open={action('dismiss')}
```

Comment:
`action('open')` etc, may want to have a `commonProps = { open: action(...) }` variable and have `{...commonProps}` here and below